src/mono/wasm/runtime/loader/libraryInitializers.ts

```diff
@@ -0,0 +1,55 @@
+import { MonoConfig, RuntimeAPI } from "../types";
+import { appendUniqueQuery, toAbsoluteBaseUri } from "./assets";
+import { normalizeConfig } from "./config";
+
+export async function fetchInitializers(moduleConfig: MonoConfig): Promise<void> {
+    if (!moduleConfig.libraryInitializers) {
+        moduleConfig.libraryInitializers = [];
+    }
+
+    const libraryInitializers = moduleConfig.resources?.libraryInitializers;
+    if (!libraryInitializers) {
+        return;
+    }
+
+    const initializerFiles = Object.keys(libraryInitializers);
+    await Promise.all(initializerFiles.map(f => importInitializer(f)));
+
+    async function importInitializer(path: string): Promise<void> {
+        const adjustedPath = appendUniqueQuery(toAbsoluteBaseUri(path));
```

Comment:
I think that try/catch/log for each of them separately would be good choice. The error should be re-thrown.

Reply:
Do mean only for import or in every export invocation?

Reply:
both could fail and should be reported.